### ISSUE TITLE
support --no-subtract-green 

### ISSUE THREAD

**Issue by matthiaskrgr** [matthiaskrgr/flifcrush#25]
from the manpage:
````
       -W, --no-subtract-green
              When specifying -Y, the fallback color transformation is RGB to  G(R-G)(B-G), i.e. the Green value gets subtracted from
              the Red and Blue channels. With this option, the green subtraction is not done.
````

**Comment by matthiaskrgr** [comment]
Currently it looks like only -W works but --no-subtract-green is missing.

**Comment by matthiaskrgr** [comment]
Fixed via a1d3d044ba9fe73762144cdc13cb3b15f9e690ed .
